packages/account/src/Sections/Verification/ProofOfIdentity/onfido.jsx

```diff
@@ -20,7 +28,15 @@ const Onfido = ({ handleRequireSubmission, is_from_external, needs_poa, onfido,
             );
         case identity_status_codes.rejected:
         case identity_status_codes.suspected:
-            if (Number(submissions_left) < 1) return <Limited handleRequireSubmission={handleRequireSubmission} />;
+            if (Number(submissions_left) < 1) {
+                return (
+                    <Unsupported
```

Comment:
@likhith-deriv is it necessary to modify Unsupported, incapsulate UploadComplete inside it and use it instead of Limited here? we can simply add the below condition to this code block, can't we? 🙂
```
if (manual && manual.status === 'pending') {
        return <UploadComplete is_from_external />;
    }
```

Reply:
Are u referring to pending status or rejected status? The requirement was to navigate user to manual upload once all the onfido submissions were used up and the docs were rejected. Earlier the user was navigated to `Contact CS via live chat` screen when the user had used up all the onfido submissions

Reply:
@likhith-deriv I referred to pending status (the code from unsupported.jsx line 30). However, now I can see that <DetailComponent /> is already using <UploadComplete/> inside it. Considering this, I wonder if it is possible to update the interface of <DetailComponent /> and the condition for it in unsupported.jsx line 35 instead of calling <UploadComplete /> directly in unsupported.jsx line 32? 🙂